src/components/form/DeleteModelForm.vue

```diff
@@ -48,18 +48,15 @@ const props = defineProps({
 const submitting = ref(false);
 
 /**
- * Delete model (folder).
+ * Delete model files.
```

Comment:
It's singular here but plural in preview commit. Probably need a change.

Reply:
use singular